target_chains/ethereum/contracts/forge-test/VerificationExperiments.t.sol

```diff
@@ -483,6 +499,121 @@ contract PythExperimental is Pyth {
         updateLatestPriceIfNecessary(priceId, info);
     }
 
+    function parseSingleAttestationFromBatch(
```

Comment:
I think it's fine to just remove it entirely from here.

Reply:
I removed `PriceAttestation` and migrated `VerificationExperiments` to use `PriceFeedMessage` instead.